main.cpp

```diff
@@ -65,31 +73,57 @@ int main()
 
 
 
+
 struct Piano
 {
-    int numKeys = 127;
-    double currentVolumeLevel = 50.0;
-    double presetVolumeLevel = 50.0;
+    Piano();
+
+    int numKeys;
+    double currentVolumeLevel;
+    double presetVolumeLevel;
     std::string pianoSound = "Grand Piano"; 
     std::string organSound = "Hammond Organ"; 
     bool footPedalPressed = false; 
 
     struct Knob
     {
-        int width = 100;
-        int height = 100;
-        float rotationAmount = 270.0f;
-        double knobValue = 0.0;
+        int width;
+        int height;
+        float rotationAmount;
+        double knobValue;
+
+        Knob(int initWidth, int initHeight, float initRotationAmount, double initKnobValue);
     };
 
     void setVolume(double volumeKnobValue);
     std::string displayChoosenSound(double soundKnobValue);
     double audio(int notePressed, double volume, bool footPedalPressed);
```

Comment:
This function name does not describe an action or verb.  
You wrote a noun for the function name

- [ ] revise